amp/system-oracle/README.md

```diff
@@ -17,6 +18,14 @@ Example:
     * instantclient-sdk-linux.x64-12.2.0.1.0.zip
     * instantclient-odbc-linux.x64-12.2.0.1.0-2.zip
 
+### Oracle Gem
+
+With the binaries in place, you'll also need to fetch the open source gem:  
+
+```
+BUNDLE_GEMFILE=Gemfile.oracle bundle package --no-install --all --path .
```

Comment:
This requires more instructions:

- install Ruby 2.4 locally
- install bundler (version 1.17.2)